src/styles/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: cente;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
```

Comment:
FYI: it's better to set it for each element on the page (use `*` selector). Note that it's the only property that is allowed to be set in `*` selector